codecov.yml

```diff
@@ -7,6 +7,11 @@ coverage:
       default: false
     patch:
       default: false
+  ignore:
+    - "Externals/*"
+    - "UnitTests/*"
```

Comment:
❗️ We don't want to exclude test code from coverage, at least not outright. Including it provides certain benefits which are otherwise difficult to impossible. It catches certain cases where tests are misbehaving but not causing a test to fail. For example:

* If a test is missing `[Test]` (shows up as an uncovered test)
* If a test contains an assertion in an asynchronous operation, but that assertion is not causing the test to fail (shows up as the line after the assertion not executing)

During the code review process, both of these can provide valuable information, and I've observed both of these cases in production code in the past and code coverage tools helped prevent a regression.

Reply:
Sounds good. Let's put it back.

One issue I thought this might resolve was the [1,569 errors in the build log](https://ci.appveyor.com/project/gitextensions/gitextensions/build/3.00.00.2350#L1038) of form:

> An System.IO.DirectoryNotFoundException occured: Could not find a part of the path 'C:\src\nunit\nunit\src\NUnitFramework\framework\Compatibility\AttributeHelper.cs'. 

Apart from causing the AppVeyor log to be truncated, this surely slows the build a bit.

I don't see these errors in builds from the last 24 hours though. Did something change, or is it specific to the build?

Reply:
I believe this was introduced by a recent upgrade of the NUnit packages. The approach seen in #4718 will correct it.

Reply:
I added filter `-[NUnit*]*` but it [didn't seem to help](https://ci.appveyor.com/project/gitextensions/gitextensions/build/3.00.00.2391). Have backed that change out.

https://github.com/OpenCover/opencover/pull/775 seems to address the problem by adjusting the log level of such messages.